### DISCUSSION TITLE
--hmr-hostname option

### DISCUSSION THREAD

**Discussion by gongevangen** [parcel-bundler/parcel#6638] [Q&A] [answered]
In Parcel v1 I uset to be able to run the server with
       parcel index.html --hmr-hostname 0.0.0.0
so I could access it using an IP address on my mobile devices.

When using --hmr-hostname on Parcel v2 I get 
      error: unknown option '--hmr-hostname'
Is there an equivalent option to --hmr-hostname on Parcel v2?

**Comment by DeMoorJasper** [comment, marked as answer]
We've simplified the cli to just have a `--host` key now that is used for both the dev server and hmr websocket. You can see all the documentation here: https://v2.parceljs.org/features/cli/

**Reply by darrentorpey** [reply]
While that reduces the footprint of the CLI options, it's also more importantly a removal of the feature where we used to be able to have HMR requests go to a different host than the regular content requests. This isn't mentioned in the Migration guide and turns out to be a sticky issue for us.

So now we just have to update our setups to not longer do this. No other workaround?

We have nginx and websocket setups with a large Docker Compose environment so changing things like this are not trivial. In our case the host the server expects to be serving as is `localhost` but the websock request needs to go to a different host in order to be routed correctly back to the dev server.

**Reply by mrcoles** [reply]
Looks like the API and the [shared docs link](https://v2.parceljs.org/features/cli/) have updated to now allow `--hmr-host` as an option?

> **--hmr-host <host>** The host for the HMR server (defaults to the dev server's host). See [Hot reloading](https://parceljs.org/features/development/#hot-reloading).
